content/nginxaas-azure/_index.md

```diff
@@ -1,11 +1,66 @@
 ---
 title: NGINXaaS for Azure
-description: 'NGINX as a Service for Azure is an IaaS offering that is tightly integrated
-  into Microsoft Azure public cloud and its ecosystem, making applications fast, efficient,
-  and reliable with full lifecycle management of advanced NGINX traffic services.
-
-  '
+nd-subtitle: Infrastructure-as-a-Service (IaaS) version of NGINX Plus for your Microsoft Azure application stack
 url: /nginxaas/azure/
+nd-landing-page: true
 cascade:
   logo: NGINX-for-Azure-icon.svg
+nd-content-type: landing-page
+nd-product: N4Azure
 ---
+
+
+## About
+NGINX as a Service for Azure is an IaaS offering that is tightly integrated
+into Microsoft Azure public cloud and its ecosystem, making applications fast, efficient,
+and reliable with full lifecycle management of advanced NGINX traffic services.
+
+## Featured content
```

Comment:
It would be great if we can list topics here based on pageviews of various sections of our docs over the past year or so. Based on customer interactions we've had, I suspect most NGINXaaS customers are looking at:
- Getting Started
- Changelog
- Logging and monitoring
- Marketplace Billing

Reply:
Just saw the message from @travisamartin and see that we do have this info which can be filtered down by products. Perhaps we can use that data to guide what goes into the featured pages section.

Reply:
added/removed cards based on feedback and tableau dashboard